src/index.ts

```diff
@@ -141,10 +141,20 @@ function jigsawBinPath() {
 /**
  * Spawns a child process to run "jigsaw build" with the proper environment.
  */
-function spawnJigsawBuild(quiet = true): Promise<void> {
+function spawnJigsawBuild(hotFile: string, quiet = true): Promise<void> {
```

Comment:
What do you think if we tweak the signature like so
```suggestion
function spawnJigsawBuild(hotFile: null | string = null, quiet = true): Promise<void> {
```

Reply:
Done!